README.md

```diff
@@ -1,13 +1,13 @@
 # Command Line Tools for Development with IBM Maximo Asset Mangement
 [![Build Status](https://travis.ibm.com/maximo-ohio/maximodev-cli.svg?token=yJyC5zQ7wEuSAyYtDC53&branch=master)](https://travis.ibm.com/maximo-ohio/maximodev-cli)
 
-The `maximodev-cli` is a set of command line tools for developing with Maximo Asset Management, which accelerates tasks, such as creating add-ons, MBOs, field validations, etc.  
+The `maximodev-cli` is a set of command line tools for developing with Maximo Asset Management that accelerates tasks, such as creating add-ons, MBOs, field validations, etc.  
 
-`maximodev-cli` requires that [NodeJS](https://nodejs.org/en/) **version 8+** be installed.  If you do not have NodeJS **version 8** installed, then install/upgrade it first.
+`maximodev-cli` requires that [NodeJS](https://nodejs.org/en/) **version 8+** be installed. If you do not have NodeJS **version 8** installed, then install or upgrade it first.
 
-The vision for `maximodev-cli` is to provide a set of lifecycle tools for the developer, allowing them to create, build, test, deploy, and package an add-on, while offering other additional tools are useful in the development lifecycle of Maximo Asset Management.
+The vision for `maximodev-cli` is to provide a set of lifecycle tools for developers, allowing them to create, build, test, deploy, and package an add-on, while offering additional tools that are useful in the development lifecycle of Maximo Asset Management.
 
-`maximodev-cli` requires direct access to your development Maximo instance's home directory.   If you are using Java support, or if you calling some commands, like `presentation-diff`, then direct access is required. Otherwise, these features will not work correctly.  You can use many features of `maximodev-cli` without a local Maximo instance, but, be warned, some tasks will fail (especially Java compiling) without local filesystem access to a Maximo instance.
+`maximodev-cli` requires direct access to your development instance of the Maximo home directory. If you are using Java™ support, or if you are calling some commands, like `presentation-diff`, then direct access is required. Otherwise, these features will not work correctly. You can use many features of `maximodev-cli` without a local Maximo instance, but, be warned, some tasks will fail, especially Java compiling, without local filesystem access to a Maximo instance.
```

Comment:
line 10
Added ™ to first reference of Java (from IBM Style: IBM will mark the Java mark by placing a superscript "TM" after Java on its first occurrence in the body of a document or web page)